### PR TITLE
Added eviction policy & updated README.md with instructions to add Budget Alerts

### PR DESCRIPTION
**Pull request by rimijoker**
## Description
- Added the feature to add eviction policy, which will tear-down the worker instances after `Cluster.sweep()` is called.
- Added instructions to setup budget alert on Google Cloud Platform.

**fixes** #3099 
@hericlesme 

## How has this been tested?
- I have created test.py and test.ipynb for testing the functionality.
- Follow the instructions in the README.md to install the dependencies.
- Run test.py or test.ipynb to test the implemented functionality.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: syft/grid/autoscale/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ In addition to a GCP account, you'll need two things to use Terraform to provisi
   - Leave the "Key Type" as JSON.
   - Click "Create" to create the key and save the key file to your system.
   - You can read more about service account keys in Google's documentation.
+  -For more info check this [link.](https://cloud.google.com/iam/docs/creating-managing-service-accounts)
```
Comment (hericlesme) [inline syft/grid/autoscale/README.md:21]: Should that be another point?

File: syft/grid/autoscale/README.md [hunk 2]
```diff
@@ -33,7 +34,21 @@ pip install terrascript
...
+
+[setup Budget and Budget Alerts](https://cloud.google.com/billing/docs/how-to/budgets)
+
+### Spin-up Instances using the follwoing commands:-
+
+You can find sample code in test .py  and  test .ipynb
+
+- Import enums from the gcloud_configurations .py
+
+```bash
```
Comment (hericlesme) [inline syft/grid/autoscale/README.md:49]: "python" would be more suitable

File: syft/grid/autoscale/README.md [hunk 3]
```diff
@@ -33,7 +34,21 @@ pip install terrascript
 
 ## Usage
 
-You can create compute instance using the sample code in test .py
+### Set Up Budget Alerts(important)
+
+Before you start to spin-up instances we encourace to set a budget alert on GCP to avoid suprise costs.
+
+[setup Budget and Budget Alerts](https://cloud.google.com/billing/docs/how-to/budgets)
```
Comment (hericlesme) [inline syft/grid/autoscale/README.md:41]: "setup" -> "Setup"

File: syft/grid/autoscale/README.md [hunk 4]
```diff
@@ -33,7 +34,21 @@ pip install terrascript
 
 ## Usage
 
-You can create compute instance using the sample code in test .py
+### Set Up Budget Alerts(important)
```
Comment (hericlesme) [inline syft/grid/autoscale/README.md:37]: Add a space between alerts and (important). Like "... Alerts (important)"

File: syft/grid/autoscale/README.md [hunk 5]
```diff
@@ -33,7 +34,21 @@ pip install terrascript
...
+
+Before you start to spin-up instances we encourace to set a budget alert on GCP to avoid suprise costs.
+
+[setup Budget and Budget Alerts](https://cloud.google.com/billing/docs/how-to/budgets)
+
+### Spin-up Instances using the follwoing commands:-
+
+You can find sample code in test .py  and  test .ipynb
```
Comment (hericlesme) [inline syft/grid/autoscale/README.md:45]: One suggestion - put the filenames like this: `test.py`.

File: syft/grid/autoscale/README.md (hunk 1 above)
Comment (rimijoker) [reply syft/grid/autoscale/README.md]: I removed this point.

**Pull request merged**